constrain/library/AppendixGHVACSystemFanOperation.py

```diff
@@ -51,30 +61,42 @@
 ```
 
 ### Data requirements
-- o: number of occupants sensed in the zones served by the system.
-- fan_runtime_fraction: system fan runtime fraction (between 0 and 1).
-- m_oa: system outdoor air flow rate.
-- tol_o: occupancy threshold; below that value the zones are considered unoccupied.
+
+- number_occupants: Number of occupants
+  - Data Value Unit: count
+  - Data Point Affiliation: Zone occupancy
+
+- fraction_runtime_fan: Fan runtime fraction
+  - Data Value Unit: fraction
+  - Data Point Affiliation: System operation
+
+- flow_volumetric_air_outdoor: Outdoor air flow rate
+  - Data Value Unit: volumetric flow rate
+  - Data Point Affiliation: System ventilation
 
 """
 
 from constrain.checklib import RuleCheckBase
 
 
 class AppendixGHVACSystemFanOperation(RuleCheckBase):
-    points = ["o", "fan_runtime_fraction", "m_oa", "tol_o"]
+    points = [
+        "number_occupants",
+        "fraction_runtime_fan",
+        "flow_volumetric_air_outdoor",
+    ]
     potential_failures_counter = 0
     potential_pass_count = 0
 
     def hvac_system_fan_operation(self, data):
-        if data["o"] >= data["tol_o"]:
-            if data["fan_runtime_fraction"] == 1:
+        if data["number_occupants"] >= self.get_tolerance("ratio", "occupancy"):
+            if data["fraction_runtime_fan"] == 1:
                 return True
             else:
                 return False
         else:
             # the system could be "cycling" for the whole timestep
-            if data["fan_runtime_fraction"] == 1:
+            if data["fraction_runtime_fan"] == 1:
```

Comment:
do we need tolerance for this? @lymereJ

Reply:
That's a good point... Not sure. I would be inclined to say no because this is a simulation focused verification but I don't feel strongly about it.

Reply:
Maybe we do need a tolerance of within 5%?